### PR TITLE
FIREFLY-1497: Remove CSS files post JoyUI

### PR DESCRIPTION
**Pull request by jaladh-singhal**
Fixes [FIREFLY-1497](https://jira.ipac.caltech.edu/browse/FIREFLY-1497)

Cleans up most (but not all) CSS files:
- 🔴 were unused, hence deleted 
- 🟥 were partially used, so replaced CSS styles by sx props and then deleted.

Following css files were present (result of `find ./src -type f -name "*.css" `):
./src/firefly/js/ui/WebApiHelpInfoPage.css - 🟥 
./src/firefly/js/ui/ProgressBar.css - 🔴
./src/firefly/js/ui/dynamic/DynamicUI.css -  🟥
./src/firefly/js/ui/panel/CollapsiblePanel.css - 🔴
./src/firefly/js/ui/ImageSelect.css - 🟥
./src/firefly/js/ui/DropDownContainer.css - 🔴
./src/firefly/js/ui/SuggestBoxInputField.css - 🟨 used in in naifId panel which should be autocomplete because dropdown is not joy-ui like - needs separate ticket
./src/firefly/js/ui/PointerPopup.css - 🔴
./src/firefly/js/ui/LinkButton.css - 🔴
./src/firefly/js/ui/VersionInfo.css - 🟥
./src/firefly/js/ui/tap/TableSelectViewPanel.css - 🟥
./src/firefly/js/ui/Banner.css - 🔴
./src/firefly/js/ui/ModalDialog.css - 🟥 + refactored to functional component
./src/firefly/js/tables/ui/TablePanel.css - 🟥 + created a new view component (a Box with sx) for .TablePanel_NoData which was being used by IFE apps (see https://github.com/IPAC-SW/irsa-ife/pull/334) + moved external style overrides to CSS-in-JS because they were using a lot of joy ui theme variables
./src/firefly/js/core/background/BackgroundMonitor.css - removed by @loitly in #1574 
./src/firefly/js/visualize/ui/ImageTableRowViewer.css -  🟥
./src/firefly/js/visualize/ui/MouseReadout.css - 🟥
./src/firefly/js/api/ApiStyle.css - 🔴

#### left unchanged:
./src/firefly/html/test/styles.css - not touching tests
./src/firefly/html/css/global.css - is used in relatively large number of files, will make diff larger and will be difficult to test - needs separate ticket (if we decide to get rid of it)

./src/firefly/js/templates/lightcurve/LCPanels.css - not touching LC template (don't know how/where it's used, is it legacy?)
./src/firefly/js/templates/fireflyslate/react-resizable_styles.css - external style overrides
./src/firefly/js/templates/fireflyslate/react-grid-layout_styles.css - external style overrides

./src/firefly/js/externalSource/prismLive/prism-live.css - external style copied and overridden
./src/firefly/js/externalSource/prismLive/prism.css - external style copied and overridden
./src/firefly/js/externalSource/FilePicker/react-keyed-file-browser/react-keyed-file-browser.css - external style copied and overridden

## Testing
https://fireflydev.ipac.caltech.edu/firefly-1497-css-removal/firefly/ - Go over each affected `*.jsx` file in diff list -> find the component affected in UI -> test that it looks the same or better

https://firefly-1497-css-removal.irsakudev.ipac.caltech.edu/applications/spherex - do any search to bring up `ImageTableRowViewer` (image slider) and check that it looks same as on nightly to test that external style overrides are intact

https://firefly-1497-css-removal.irsakudev.ipac.caltech.edu/applications/ztf - do any search that yields empty charts tab and/or details tab, `NoDataTableView` should look ok

---
Create ticket for leftovers (if we decide to change them?)

### CODE REVIEW

**Comment by gpdf** [comment]
Is this aimed at 2024.2.x or 2024.3?

**Review by loitly — APPROVED** [review]
I did a little testing and it looks good.

**Review by robyww — APPROVED** [review]
Changes look good.

File: src/firefly/js/visualize/ui/CatalogConstraintsPanel.jsx [hunk 1]
```diff
@@ -393,7 +393,7 @@ class ConstraintPanel extends PureComponent {
                 '& .fixedDataTableLayout_main': {borderRadius:'5px'} }}>
                 <Stack {...{ direction:'row', flex: '1 1 auto', position: 'relative', width: 1, height: 1}}>
                     <Box sx={{minHeight:150, flex: '1 1 auto'}}>
-                            <div className='TablePanel__table' style={{top: 0}}>
+                            <Stack sx={{m: '1px', position: 'absolute', top: 0, bottom: 0, left: 0, right: 0}}>
```
[the remark this answers is not in the document]
Comment (jaladh-singhal) [reply src/firefly/js/visualize/ui/CatalogConstraintsPanel.jsx:395]: Good idea.

**Pull request merged**